aws-greengrass-testing-features/aws-greengrass-testing-features-api/src/main/java/com/aws/greengrass/testing/features/FileSteps.java

```diff
@@ -114,6 +115,29 @@ public void containsTimeout(String file, String contents, int value, String unit
         }
     }
 
+    /**
+     * File contains content at least N times after a duration.
+     *
+     * @param file file on a {@link Device}
+     * @param contents file contents
+     * @param times how many times contents should be present in file
+     * @param value integer value for a duration
+     * @param unit {@link TimeUnit} duration
+     * @throws InterruptedException thread was interrupted while waiting
+     */
+    @Then("the file {word} on device contains {string} at least {int} times within {int} {word}")
```

Comment:
this change is not related to network utils. Please split unrelated changes into different PRs.

Reply:
Moved to https://github.com/aws-greengrass/aws-greengrass-testing/pull/214

Reply:
removed in d98e9434740bb90c6989c794c9d58e006903b9e1